src/main/java/it/gov/pagopa/pu/sil/mapper/ClassificationsExportFileRequestMapper.java

```diff
@@ -21,9 +22,12 @@ public ClassificationsExportFileRequestDTO mapToExportFileRequest(Long organizat
       .exportFileType(ClassificationsExportFileRequestDTO.ExportFileTypeEnum.CLASSIFICATIONS)
       .fileVersion(request.getVersioneTracciato())
       .filterFields(new ClassificationsExportFileFilter()
+        .debtPositionTypeOrgCodes(new HashSet<>(request.getTipoDovuto().getTipos()))
         .label(request.getCodiceClassificazione().getClassificaziones().stream()
           .map(ClassificationsExportFileFilter.LabelEnum::fromValue)
           .collect(Collectors.toSet()))
+        .iuv(request.getIdUnivocoVersamento().getIuvs())
+        .iur(request.getIdUnivocoRendicontazione().getIurs())
         .iud(request.getIdUnivocoDovuto())
```

Comment:
are these property that you are reading required? otherwise you will go into nullpointerException reading them ASIS